react-hot-button-component/src/index.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+
+class CustomButton extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isClicked: false };
```

Comment:
In state you should be storing the number of clicks, `isClicked` isn't useful for this exercise.